### PR TITLE
Refactor reorg

### PR DESCRIPTION
**Pull request by bobmcwhirter**

### CODE REVIEW

**Comment by bobmcwhirter** [comment]
Do not merge

File: lib/src/client/certify_vuln/ingest.rs [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+
+#[cfg(test)]
+pub mod test {
+    use crate::client::certify_vuln::Ghsa;
+    use crate::client::GuacClient;
+
+    use super::*;
+
+    #[tokio::test]
+    async fn ingest_vuln() -> Result<(), anyhow::Error> {
```
Comment (dejanb) [inline lib/src/client/certify_vuln/ingest.rs:113]: This test fails with `message: "IngestVulnerability ::  input: ghsa id \"ghsa-taco-vuln\" not found"`

**Review by dejanb — COMMENTED** [review]
Looks good in general. I added some code formatting and there is also one test failing. I guess it's time now to add some github actions for testing things.

**Review by dejanb — APPROVED** [review]
Looks good. Trustification compiles and basic tests are passing.

**Pull request merged**